### DISCUSSION TITLE
%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA

### DISCUSSION THREAD

**Discussion by giscus** [Jackiexiao/nolebase-template#1] [Announcements]
# %E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA

记录回忆，知识和畅想的地方

https://nolebase-template.vercel.app/%E7%AC%94%E8%AE%B0/%E6%BC%94%E7%A4%BA%E7%9B%AE%E5%BD%951/%E6%96%87%E6%A1%A3-%E5%B8%A6%E8%AF%84%E8%AE%BA.html

**Comment by makerjackie** [comment]
测试评论

**Comment by lesswrongzh** [comment]
这是一个评论
